tests/puppeteer/campaigns/data/demo/tax.js

```diff
@@ -1,8 +1,9 @@
 module.exports = {
   DefaultFrTax: {
-    id: '1',
+    id: 1,
     name: 'TVA FR 20%',
-    rate: '20',
+    frName: 'FR Taux standard (20%)',
```

Comment:
Why not have an array with `fr` and `en` as keys ?

Reply:
As we said [here](https://github.com/PrestaShop/PrestaShop/pull/18426#discussion_r402114526), we don't need an array as there is only 2 keys.
And for other countries taxes, maybe there is only one key or maybe 2 keys ('en, 'Not fr').
For now, I think no need to break or change a lot of tests, but in the future, if this must be done, we will do it.

Reply:
In this case, we need to rename name in enName

Reply:
I have a solution, since this value `frName: 'FR Taux standard (20%)'` is a one time use, I will delete it.

Reply:
the FrName was deleted from demo tax Since it's one time use. Thanks @Progi1984